source/connection-troubleshooting.txt

```diff
@@ -159,10 +159,11 @@ Connection Troubleshooting
 Security Certificate Errors
 ---------------------------
 
-If you use Java version 8 or earlier, you might need to add a certificate to
-your trust store. You can either upgrade to a later version of the JDK or read
-the :atlas:`Security FAQ </reference/faq/security/#java-users>` instructions
-in the Atlas documentation for information on how to add the certificate.
+If you use Java version 8 or earlier, you might be required to add a
```

Comment:
Q: just for my own edification--are we supposed to avoid "need to"?

Edit: Looks like we should, but only in the interest of [brevity.](https://www.mongodb.com/docs/meta/style-guide/terminology/concise-terms/#:~:text=usually-,need%20to,-must) In this case, "need to" is actually shorter!